Src/DependencyCollector/Shared/Implementation/ProfilerSqlProcessingBase.cs

```diff
@@ -92,6 +92,24 @@ public object OnEndForOneParameter(object context, object returnValue, object th
         }
 
         /// <summary>
+        /// On end async callback for methods with 1 parameter.
+        /// </summary>
+        public object OnEndAsyncForOneParameter(object context, object returnValue, object thisObj)
```

Comment:
Is not used now, but will be when we'll update SQL async methods instrumentation.

---
In reply to: [98060621](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/298#discussion_r98060621) [](ancestors = 98060621)